### ISSUE TITLE
[SCREENREADER]: Progress Button - Users SHOULD NOT hear the before and after arrows announced

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10438]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our [Progress Button component](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/progressbutton/) has the ability to add before and after text, which we're usually using forward and backward arrows. I'd like to hide this text from screen readers because it's announced as "Continue, double right angle bracket, button" which could be confusing. Code snippet below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Screen readers ignore the visual text flourishes like double angle brackets before or after button text
- [ ] No axe violations

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/progressbutton/

## Possible Fixes (optional)

```diff
render() {
  const beforeText = this.props.beforeText ? (
-   <span className="button-icon">{this.props.beforeText} </span>
+   <span aria-hidden="true" className="button-icon">{this.props.beforeText} </span>
  ) : (
    ''
  );
  const afterText = this.props.afterText ? (
-   <span className="button-icon"> {this.props.afterText}</span>
+   <span aria-hidden="true" className="button-icon"> {this.props.afterText}</span>
  ) : (
    ''
  );
  ... more render data
}
```

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #220](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/220) via [**ZenHub**](https://www.zenhub.com/)
